packages/react/modules/components/configure-adapter/with-reconfiguration.spec.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import withReconfiguration from './with-reconfiguration';
+
+const TestComponent = () => <div />;
+TestComponent.displayName = 'TestComponent';
+
+describe('rendering', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    const Component = withReconfiguration('foo-prop-key')(TestComponent);
```

Comment:
You probably want to test it also without the custom prop name. Also to check that the wrapped component gets the `reconfigure` prop?